src/ring_theory/local_properties.lean

```diff
@@ -99,10 +100,43 @@ if `P` holds for `R →+* S` whenever there exists a set `{ r }` that spans `R`
 Note that this is equivalent to `ring_hom.of_localization_finite_span` via
 `ring_hom.of_localization_span_iff_finite`, but this has less restrictions when applying. -/
 def ring_hom.of_localization_span :=
-  ∀ {R S : Type u} [comm_ring R] [comm_ring S] (f : by exactI R →+* S)
+  ∀ ⦃R S : Type u⦄ [comm_ring R] [comm_ring S] (f : by exactI R →+* S)
     (s : set R) (hs : by exactI ideal.span s = ⊤)
     (H : by exactI (∀ (r : s), P (localization.away_map f r))), by exactI P f
 
+/-- A property `P` of ring homs satisfies `ring_hom.localization_away_is`
+ if `P` holds for each localization map `R →+* Rᵣ`. -/
+def ring_hom.localization_away_is : Prop :=
+∀ ⦃R : Type u⦄ (S : Type u) [comm_ring R] [comm_ring S] [by exactI algebra R S] (r : R)
+  [by exactI is_localization.away r S], by exactI P (algebra_map R S)
+
+/-- A property `P` of ring homs satisfies `ring_hom.of_localization_span_target`
+if `P` holds for `R →+* S` whenever there exists a finite set `{ r }` that spans `S` such that
+`P` holds for `R →+* Sᵣ`.
+
+Note that this is equivalent to `ring_hom.of_localization_span` via
+`ring_hom.of_localization_span_iff_finite`, but this is easier to prove. -/
```

Comment:
Are you planning to prove this equivalence? If so I'd rather see `of_localization_span` in `property_is_local` plus a theorem stating the equivalence of these conditions.

Reply:
I don't think the equivalence is true. I probably just copy pasted stuff without thinking :(